gridplayer/multiprocess/process_manager.py

```diff
@@ -71,11 +71,14 @@ def active_instances(self):
                         active_instances.append(instance)  # noqa: WPS220
         return active_instances
 
-    def get_instance(self):
-        instance = self._get_available_instance()
+    def get_instance(self, options=[]):
```

Comment:
I don't like that we'll have to generate a new instance on every options change, but grouping instances by options doesn't seem like a big improvement. So I'll leave it like you suggested.

Reply:
Yeah, grouping would be more tidy. 

On the other hand, it should be ok if instances are created and terminated correctly. Actually, when I test gridplayer playing cameras live streams, I found that setting "Videos per process" to 1 give more stable performance. A typical case is the instance stuck at processing a video, then other videos (using the same instance) need to wait.